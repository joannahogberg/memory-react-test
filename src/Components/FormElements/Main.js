import React from 'react';

const Main = (props)=>{
    return(
	<main className="flex flex-col justify-center items-center w-full mx-auto font-sans text-indigo-darker mt-6">
		{props.children}
	</main>
	); 
}

export default Main;