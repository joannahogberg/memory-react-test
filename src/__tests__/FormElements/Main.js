import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow, render } from 'enzyme';
import Main from '../../Components/FormElements/Main.js';

it('Check snapshot for Main', () => {
    const wrapper = render(<Main />);
    expect(toJSON(wrapper))
  .toMatchSnapshot("Main snap");
})

it('Check snapshot for Main', () => {
    const wrapper = shallow(<Main />);
   expect(wrapper.props().className).toContain('justify-center')
})