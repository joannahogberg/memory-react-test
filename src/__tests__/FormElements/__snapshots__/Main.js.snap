// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main snap 1`] = `
<main
  class="flex flex-col justify-center items-center w-full mx-auto font-sans text-indigo-darker mt-6"
>
  
</main>
`;
